extensions/podman/src/macos-checks.ts

```diff
@@ -67,23 +66,28 @@ export class MacPodmanInstallCheck extends BaseCheck {
   title = 'Podman Installation';
   async execute(): Promise<extensionApi.CheckResult> {
     // we need to check if brew is installed to avoid unexpected error
-    const brewResult = await runCliCommand('which', ['brew']);
-    if (brewResult.exitCode !== 0) {
-      // brew is not installed so do not check if podman has been installed with brew
-      return this.createSuccessfulResult();
+    try {
+      await extensionApi.process.exec('which', ['brew']);
+    } catch (err) {
+      if ((err as extensionApi.RunError) && err.exitCode !== 0) {
+        // brew is not installed so do not check if podman has been installed with brew
+        return this.createSuccessfulResult();
+      }
     }
 
     // brew is installed, check if podman has been installed with brew
-    const runResult = await runCliCommand('brew', ['list', '--verbose', 'podman'], {
-      env: { HOMEBREW_NO_AUTO_UPDATE: '1', HOMEBREW_NO_ANALYTICS: '1' },
-    });
-
-    if (runResult.exitCode === 0) {
-      return this.createFailureResult(
-        'You have podman installed with "brew", run "brew update && brew upgrade podman" to install new version',
-        'Brew Documentation',
-        'https://docs.brew.sh/Manpage#upgrade-options-outdated_formulaoutdated_cask-',
-      );
+    try {
+      await extensionApi.process.exec('brew', ['list', '--verbose', 'podman'], {
+        env: { HOMEBREW_NO_AUTO_UPDATE: '1', HOMEBREW_NO_ANALYTICS: '1' },
+      });
+    } catch (err) {
+      if ((err as extensionApi.RunError) && err.exitCode !== 0) {
+        return this.createFailureResult(
+          'You have podman installed with "brew", run "brew update && brew upgrade podman" to install new version',
+          'Brew Documentation',
+          'https://docs.brew.sh/Manpage#upgrade-options-outdated_formulaoutdated_cask-',
+        );
+      }
```

Comment:
agree with @lstocchi 

the error here is if we see that podman is installed using brew that we need to report an error
(so basically if no error then create an error)